File: mvvmbase/src/main/java/com/bing/mvvmbase/bindingadapter/BindingAdapters.java
package com.bing.mvvmbase.bindingadapter;

import android.view.View;
import android.widget.ImageView;

import com.bing.mvvmbase.module.glide.GlideApp;

import androidx.databinding.BindingAdapter;

public class BindingAdapters {
	@BindingAdapter("visibleGone")
	public static void showHide(View view, boolean show) {
		view.setVisibility(show ? View.VISIBLE : View.GONE);
	}

	@BindingAdapter("img_url")
	public static void showImg(ImageView iv, String url) {
		GlideApp.with(iv.getContext()).load(url).into(iv);
	}
}

File: mvvmbase/src/main/java/com/bing/mvvmbase/bindingadapter/BindingAdapters.kt
package com.bing.mvvmbase.bindingadapter

import android.view.View
import android.widget.ImageView

import androidx.databinding.BindingAdapter
import com.bumptech.glide.Glide

@BindingAdapter("visibleGone")
fun showHide(view: View, show: Boolean) {
        view.visibility = if (show) View.VISIBLE else View.GONE
}

@BindingAdapter("img_url")
fun showImg(iv: ImageView, url: String) {
        Glide.with(iv.context).load(url).into(iv)
}
